VideoLocker/res/layout/fragment_course_combined_info.xml

```diff
@@ -1,86 +1,73 @@
 <ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
-            xmlns:app="http://schemas.android.com/apk/res-auto"
-            xmlns:tools="http://schemas.android.com/tools"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:background="@color/edx_grayscale_neutral_white"
-            android:fillViewport="true"
-            tools:context="org.edx.mobile.view.CourseCombinedInfoFragment">
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/edx_grayscale_neutral_white"
+    android:fillViewport="true"
+    tools:context="org.edx.mobile.view.CourseCombinedInfoFragment">
 
     <LinearLayout
+        android:id="@+id/announcement_layout"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+        android:layout_height="match_parent"
```

Comment:
The child should be wrap_content. Android studio warns about this, you should see it when looking at the file.